packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -64,6 +64,8 @@ export interface FieldDisplay {
   rowIndex?: number; // only filled in when the value is from a row (ie, not a reduction)
   getLinks?: () => LinkModel[];
   hasLinks: boolean;
+  fieldType?: FieldType;
```

Comment:
remove `fieldType` you can get it from `sourceField.type` :)

Reply:
LOL! Of course, sorry. On it.